packages/filebrowser/style/base.css

```diff
@@ -149,6 +153,12 @@
   text-align: right;
 }
 
+.jp-DirListing-headerItem.jp-id-filesize {
+  flex: 0 0 70px;
```

Comment:
```suggestion
  flex: 0 0 75px;
```

`72px` was the minimum for me to avoid this problem:

![Screenshot from 2023-02-21 21-22-16](https://user-images.githubusercontent.com/5832902/220462104-a315d0a3-36e8-4963-a913-51a25d4a5342.png)

but I suggest adding 3 more pixels to be on the safe side.

Reply:
(happy to meet in the middle)